Python/Music_Genre_Classification/feature_extraction.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Imports
+
+import librosa
+import numpy as np
+
+import os
+import csv
+
+# The Features to be Extracted as headers
+
+header = 'filename chroma_stft rmse spectral_centroid spectral_bandwidth rolloff zero_crossing_rate'
+for i in range(1, 21):
+    header += f' mfcc{i}'
+header += ' label'
+header = header.split()
+
+# Creating a CSV File
+
+file = open('GTZAN.csv', 'w', newline='')
+with file:
+    writer = csv.writer(file)
+    writer.writerow(header)
+
+# The Genres in Dataset
+genres = 'blues classical country disco hiphop jazz metal pop reggae rock'.split()
+
+for g in genres:
+    # Traversing through various genres in the Dataset
+    # Feed the complete path of the GTZAN folder
+    for filename in os.listdir(f'Database/GTZAN/{g}'):
+
+        # Traversing through various songs in a particular genre.
+        songname = f'C:/Users/asus/PycharmProjects/Music_Classification/Database/GTZAN/{g}/{filename}'
```

Comment:
Can you remove all local references? It is preferred to use absolute paths

Reply:
What's wrong with me, I have been pushing a lot of local addresses lately, it skips my mind for some reason 😞

Reply:
No worries! You are doing a great work.